galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/TestRunner.java

```diff
@@ -390,7 +391,7 @@ public void runTest(Properties bootstrapProperties, Properties overridePropertie
                         logger.error("Problem cleaning shared environment properties", e);
                     }
                 }
-                updateStatus("finished", "finished");
+                updateStatus(statuses.finished.toString(), statuses.finished.toString());
```

Comment:
Shouldn't updateStatus take a status rather than a string now that we have an enum ?